app/views/catalog_record_ids/_edit.html.erb

```diff
@@ -24,17 +24,21 @@
         <% end %>
 
         <template data-nested-form-target='template'>
-          <%= form.fields_for :catkeys, CatkeyForm::Row.new, child_index: 'TEMPLATE_RECORD' do |catkey_form| %>
-            <%= render 'row', form: catkey_form %>
+          <%= form.fields_for :catalog_record_ids, CatalogRecordIdForm::Row.new, child_index: 'TEMPLATE_RECORD' do |catalog_record_id_form| %>
+            <%= render 'row', form: catalog_record_id_form, hidden: false %>
           <% end %>
         </template>
 
-        <%= form.fields_for :catkeys do |catkey_form| %>
-          <%= render 'row', form: catkey_form %>
+        <%= form.fields_for :catalog_record_ids do |catalog_record_id_form| %>
+          <%= render 'row', form: catalog_record_id_form, hidden: false %>
+        <% end %>
+
+        <%= form.fields_for :other_catalog_record_ids do |catalog_record_id_form| %>
+          <%= render 'row', form: catalog_record_id_form, hidden: true %>
         <% end %>
```

Comment:
Added this to help persist Folio catalog links when in Symphony mode, and vice versa.